src/browser/modules/Stream/ParamsFrame.tsx

```diff
@@ -47,14 +47,14 @@ const ParamsFrame = ({ frame }: any) => {
     typeof frame.success === 'undefined' ? null : (
       <StyledStatsBar>
         <Ellipsis>
-          <Render if={frame.success === true}>
```

Comment:
just a nit, but this could be combined into ternary, no?

Reply:
I wasn't confident with this, with the weird "undefined" string check, so leaving it